_artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-rpm.markdown

```diff
@@ -0,0 +1,13 @@
+---
+role: daemon
+artifact_id: opensearch
+version: 2.0.0-rc1
+platform: linux
+architecture: arm64
+artifact_url: https://artifacts.opensearch.org/releases/bundle/opensearch/2.0.0-rc1/opensearch-2.0.0-rc1-linux-arm64.rpm
```

Comment:
The version and URL for this file will need to be changed. 
https://artifacts.opensearch.org/releases/bundle/opensearch/1.3.2/opensearch-1.3.2-linux-x64.rpm

Reply:
fixed